server/services/DashboardService/index.js

```diff
@@ -0,0 +1,132 @@
+import { createHash } from 'crypto'
+import _ from 'lodash'
+import CollectionsModel from '../../models/CollectionsModel'
+import MetadataModel from '../../models/MetadataModel'
+
+class DashboardService {
+  constructor(dataDb, study, subject, configuration) {
+    this.assessmentKey = 'analysis'
+    this.collection = 'collection'
+    this.configDataMap = new Map()
+    this.configuration = configuration
+    this.consentDate = ''
+    this.consentDateKey = 'Consent Date'
```

Comment:
do we need all these attributes on the service or are the unchanging constants?

Reply:
e26508b819e80caaf8bc6d068855562cd17fdc07